docs/manuals/rest/index.md

```diff
@@ -48,6 +48,9 @@ To expose the Taipy REST APIs, the Taipy REST server must first be started.
    * Running on http://127.0.0.1:5000 (Press CTRL+C to quit)
    ```
 
+!!! Note "When running Taipy Rest server, you will also run `Core^`"
+!!! info "For running Taipy Rest service with the other Taipy services, please refer to page [Running Taipy services](../running_services/index.md)"
```

Comment:
"For running Taipy Gui service"
->
"To run the Taipy GUI service"

"to page [Running Taipy services](https://github.com/Avaiga/taipy-doc/pull/running_services/index.md)"
to
"to the [Running Taipy services](https://github.com/Avaiga/taipy-doc/pull/running_services/index.md) page"